recipes/hyperas/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hyperas" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/maxpumperla/hyperas/archive/{{ version }}.tar.gz"
+  sha256: "2f82ad10dd37375330462a770fcdffc45aca735ca9dd5dc54f695fd48300bc5b"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - entrypoints
+    - hyperopt
+    - jupyter
+    - keras
+    - nbconvert
+    - nbformat
+    - pip
+    - python
+  run:
+    - entrypoints
+    - hyperopt
+    - jupyter
+    - keras
+    - nbconvert
+    - nbformat
+    - python
+
+test:
+  imports:
+    - hyperas
+
+about:
+  home: "http://github.com/maxpumperla/hyperas"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "Simple wrapper for hyperopt to do convenient hyperparameter optimization for Keras models"
+  dev_url: "http://github.com/maxpumperla/hyperas"
+
+extra:
+  recipe-maintainers:
+    - marcelotrevisani
+    - Tata17
```

Comment:
Could you please confirm that you want to maintain this recipe as well?

Reply:
I can be a maintainer, thanks!